src/partitions.jl

```diff
@@ -184,7 +187,7 @@ function nextsetpartition(s::AbstractVector, a, b, n, m)
             end
         end
         a[j] += 1
-        m = b[j] + (a[j] == b[j])
+        m = b[j] + Int(a[j] == b[j])
```

Comment:
`m` is initially of type `Int64` and `b[j]` is of type `Int32`. Should I alternatively change `b` to `Vector{Int64}` or `m` to `Int32`?